pkg/ui/react-app/package.json

```diff
@@ -35,6 +35,7 @@
     "downshift": "^6.1.0",
     "enzyme-to-json": "^3.6.1",
     "fuzzy": "^0.1.3",
+    "hosted-git-info": "2.8.9",
```

Comment:
This is not a direct dependency so we should not add it in the `package.json` file